### ISSUE TITLE
check NASA data for precipitation

### ISSUE THREAD

**Issue by mostaphaRoudsari** [mostaphaRoudsari/ladybug#86]
Thanks to HKS guys:

http://www.nasa.gov/press/2014/september/international-global-precipitation-measurement-mission-data-goes-public/#.VCV_ovldXE1

http://www.nasa.gov/mission_pages/GPM/main/index.html#.VCV_oPldXE0

http://pmm.nasa.gov/data-access/downloads/gpm

**Comment by chriswmackey** [comment]
Panos is looking into this now.  Given the satellite-based nature of the dataset, we might be better able to parse it with the functions that I am building in Dragonfly.  This can be discussed more after I get the basic functionality of DF set and Panos has gotten a chance to understand the dataset.

**Comment by chriswmackey** [comment]
This still remains a relevant issue and I welcome anyone to tackle it.  Also, anyone who tackles this issue should also check the progress made through this discussion: http://discourse.ladybug.tools/t/rainfall-data-component/2197

**Comment by AntoineDao** [comment]
Does the PR above resolve this @chriswmackey ?

**Comment by mostaphaRoudsari** [comment]
This hasn't been resolved ans is a pretty good issue for hacktoberfest.

**Comment by devngc** [comment]
@mostaphaRoudsari,
The component shared in the last post by Rafael seems to work. Do you have any comments on what Rafael put together?

**Comment by mostaphaRoudsari** [comment]
@devngc, I haven't tested it not sure if @chriswmackey has done the tests. If it works then one should send a PR.

**Comment by chriswmackey** [comment]
Agreed. Send a PR and we will be very happy to comment on anything that should be changed and then merge it.

**Comment by SpaceArchitecture** [comment]
Any updates on this component ?
